app-kotlin/build.gradle

```diff
@@ -65,11 +66,21 @@ dependencies {
 
 //  Kotlin Coroutines
   implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.3'
+  testImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.3'
 
-//  picasso
+//  Picasso
   implementation 'com.squareup.picasso:picasso:2.8'
 
 //  Koin for Kotlin apps
   implementation 'io.insert-koin:koin-core:3.2.0'
   implementation 'io.insert-koin:koin-android:3.2.0'
+
+//  Koin testing tools
+  testImplementation "io.insert-koin:koin-test:3.2.0"
+//  Needed JUnit version
+  testImplementation "io.insert-koin:koin-test-junit4:3.2.0"
+
+//  Mockk Test
+  testImplementation "io.mockk:mockk:1.12.4"
+  androidTestImplementation "io.mockk:mockk-android:1.12.4"
```

Comment:
do you need this?

Reply:
duplicated we need one of this of mocking service